homeassistant/components/fritz/update.py

```diff
@@ -55,12 +50,12 @@ def __init__(
         self,
         avm_wrapper: AvmWrapper,
         device_friendly_name: str,
-        description: FritzUpdateEntityEntityDescription,
     ) -> None:
         """Init FRITZ!Box connectivity class."""
-        self.entity_description = description
-        self._attr_name = f"{device_friendly_name} {description.name}"
-        self._attr_unique_id = f"{avm_wrapper.unique_id}-{description.key}"
+        self.entity_description = ENTITY_DESCRIPTION
+        self._attr_title = f"{device_friendly_name}"
```

Comment:
`device_friend_name` is the entry title, which is not what this title is meant for.
The `title` attribute of an update entity provides the name of the software that has a pending update (which is different from e.g., an entry title or entity name)

Reply:
@frenck 
Ah I think I got it now (hopefully). Changed the title just to the name of the os and dropped everythink with "description" since this is just the update entity for the configured device. 
Thanks for your patience :)